rest-service/manager_rest/config.py

```diff
@@ -50,6 +50,7 @@ def __init__(self):
         self._security_authentication_providers = []
         self._security_authorization_provider = None
         self._insecure_endpoints_disabled = False
+        self._security_agent_rest_credentials = None
```

Comment:
having one property with username/password is inconsistent with other properties that are top level.
while it makes sense generally, i don't think this is how it should be implement to keep up with current implementation.